### ISSUE TITLE
Locations MC&IC and IC&MC

### ISSUE THREAD

**Issue by T3hlo** [AmsterdamUMC/AmsterdamUMCdb#58]
There are four different locations in the admission table as stated in the [wiki](https://github.com/AmsterdamUMC/AmsterdamUMCdb/wiki/admissions) they are as follows:
`the department the patient has been admitted to, either IC, MC or both (IC&MC or MC&IC)`

I have two question about these locations:

My first question is does the order of the ward acronyms indicate the patients journey. E.g. did a IC&MC move from an IC ward to an MC ward or are they interchangeble? 

I have noticed that the combined locations (IC&MC or MC&IC) are a newer location description the admission to these wards have increased after 2010 see the admission counts by ward and year below.

![Screenshot from 2022-03-28 14-30-15](https://user-images.githubusercontent.com/45871693/160398673-1add7109-2883-4502-a928-2a41e8edec19.png)

So the second question is are these physically new wards that were added to the hospital at some point?

**Comment by patrickthoral** [comment]
You are correct that `MC&IC` means that the patient started on the `Medium Care` unit. The `Medium Care` unit initially did not use the electronic system of the ICU (MetaVision), so that explains why that information is not available for all patients in the period from 2003-2009.
